src/prefect/task_runners.py

```diff
@@ -65,6 +65,7 @@
 from uuid import UUID
 
 import anyio
+from anyio import CapacityLimiter
```

Comment:
Can you use `anyio.CapacityLimiter` instead of importing the name? It's nice to have the clarity of where the implementation is.